frontend/js/src/playlists/UserSearch.tsx

```diff
@@ -0,0 +1,96 @@
+import * as React from "react";
+import { faUser } from "@fortawesome/free-solid-svg-icons";
+import { throttle as _throttle } from "lodash";
+import debounceAsync from "debounce-async";
+import GlobalAppContext from "../utils/GlobalAppContext";
+import ListenControl from "../listens/ListenControl";
+
+export type UserSearchProps = {
+  userClick: (event: string) => void;
```

Comment:
What does this do? Is it this?
```suggestion
  onSelectUser: (userName: string) => void;
```